src/client/CourseRequestForm/index.js

```diff
@@ -223,18 +229,18 @@ const CourseRequestForm = ({ courses }) => {
     ))
   );
   const courseRequestFormHeaderList = [
-    'Request ID',
+    // 'Request ID',
```

Comment:
@tatikamsetti Why is this commented out? For new requests, it should just be blank like it is currently on CCLE

Reply:
@rlorenzo As the ticket says... it should match the screenshot attached and I do not see Request Id in the header list. Also to confirm, I looked at ccle's course requestor and deleted it.
See the CCLE's course requestor form:
![Screen Shot 2020-12-02 at 6 34 15 PM](https://user-images.githubusercontent.com/12834134/100956325-1595a200-34cd-11eb-9126-78b3c959e1fc.png)

I can add back if you want me to.